xbmc/pvr/addons/PVRClient.cpp

```diff
@@ -308,58 +310,7 @@ bool CPVRClient::GetAddonProperties()
 
   if (retVal != PVR_ERROR_NO_ERROR)
     return false;
-
-  /* get the name of the backend */
-  retVal = DoAddonCall(
-      __func__,
-      [&strBackendName](const AddonInstance* addon) {
-        return addon->toAddon->GetBackendName(addon, strBackendName, sizeof(strBackendName));
-      },
-      true, false);
-
-  if (retVal != PVR_ERROR_NO_ERROR)
-    return false;
-
-  /* get the connection string */
-  retVal = DoAddonCall(
-      __func__,
-      [&strConnectionString](const AddonInstance* addon) {
-        return addon->toAddon->GetConnectionString(addon, strConnectionString,
-                                                   sizeof(strConnectionString));
-      },
-      true, false);
-
-  if (retVal != PVR_ERROR_NO_ERROR && retVal != PVR_ERROR_NOT_IMPLEMENTED)
-    return false;
-
-  /* display name = backend name:connection string */
-  strFriendlyName = strlen(strConnectionString) == 0 // connection string is optional
-                        ? strBackendName
-                        : StringUtils::Format("{}:{}", strBackendName, strConnectionString);
-
-  /* backend version number */
-  retVal = DoAddonCall(
-      __func__,
-      [&strBackendVersion](const AddonInstance* addon) {
-        return addon->toAddon->GetBackendVersion(addon, strBackendVersion,
-                                                 sizeof(strBackendVersion));
-      },
-      true, false);
-
-  if (retVal != PVR_ERROR_NO_ERROR)
-    return false;
-
-  /* backend hostname */
-  retVal = DoAddonCall(
-      __func__,
-      [&strBackendHostname](const AddonInstance* addon) {
-        return addon->toAddon->GetBackendHostname(addon, strBackendHostname,
-                                                  sizeof(strBackendHostname));
-      },
-      true, false);
-
-  if (retVal != PVR_ERROR_NO_ERROR && retVal != PVR_ERROR_NOT_IMPLEMENTED)
-    return false;
+  const std::string strFriendlyName = GetFriendlyName();
```

Comment:
This variable can be avoided:

```diff
diff --git a/xbmc/pvr/addons/PVRClient.cpp b/xbmc/pvr/addons/PVRClient.cpp
index b44da852bd..d5c4947149 100644
--- a/xbmc/pvr/addons/PVRClient.cpp
+++ b/xbmc/pvr/addons/PVRClient.cpp
@@ -310,12 +310,11 @@ bool CPVRClient::GetAddonProperties()
 
   if (retVal != PVR_ERROR_NO_ERROR)
     return false;
-  const std::string strFriendlyName = GetFriendlyName();
 
   /* timer types */
   retVal = DoAddonCall(
       __func__,
-      [this, strFriendlyName, &addonCapabilities, &timerTypes](const AddonInstance* addon) {
+      [this, &addonCapabilities, &timerTypes](const AddonInstance* addon) {
         std::unique_ptr<PVR_TIMER_TYPE[]> types_array(
             new PVR_TIMER_TYPE[PVR_ADDON_TIMERTYPE_ARRAY_SIZE]);
         int size = PVR_ADDON_TIMERTYPE_ARRAY_SIZE;
@@ -328,7 +327,7 @@ bool CPVRClient::GetAddonProperties()
           CLog::LogF(LOGWARNING,
                      "Add-on {} does not support timer types. It will work, but not benefit from "
                      "the timer features introduced with PVR Addon API 2.0.0",
-                     strFriendlyName);
+                     GetFriendlyName());
 
           // Create standard timer types (mostly) matching the timer functionality available in Isengard.
           // This is for migration only and does not make changes to the addons obsolete. Addons should
```